### PR TITLE
Removed speed constraints from CRZ speed

### LINKED ISSUE #3013: [BUG] Speed constraints erroneously in MCDU en route and slowing A/C at CRZ FL.
**Mod Version**
Latest Custom FBW (D/L 12-Jan AM)

**Describe the bug**
When at CRZ, MCDU shows speed constraints at some waypoints mid-journey and A/C erroneously slows to match.  

Example: Mid-flight en route from KORD to KLAS, used following: DCT VORIN DCT IOW DCT CNOTA DCT LNK DCT MCK DCT FQF DCT SINRY DCT CHESZ DCT GGAPP SITEE4, CRZ FL360.  

At SINRY (middle of route), MCUD displayed speed constraint of 250K, while managed CRZ was 290.  A/C automatically slowed to 250K prior to SINRY and automatically increased back to 290K after passing CHESZ.

Again, at GGAPP where MCDU indicated 210K, A/C dropped to 210K mid cruise once passing.  Had to use speed hold mode from thence.  Descent wasn't to begin until much later.

**To Reproduce**
1. Plan flight using IFR/High Alt Option.
2. Load A/C cold and dark and go through pre-flight MCDU check of FPL.
3. Note any constraints mid-flight (cannot be cleared).
4. Fly planned route and pay attention at noted waypoints.

**Expected behavior**
At CRZ, A/C should maintain planned speed for CRZ until descend mode is activated, regardless of waypoints below and any constraints that may apply at such points as a part of other STAR/SID routes.  No speed constraints should load on ANY waypoints during planned route (after departure, before arrival/approach).

**Actual behavior**
A/C changes speed to match constraints in MCDU which cannot be cleared/erased.

**References**
N/A

**Additional context**
N/A

Was this working before/when did the issue start occurring?
Prior to using updated version, this was not an issue.  I believe these changes were implemented a few days ago, right around the 1/7/21 MSFS update.

Is this a problem in the vanilla unmodded game?
Doubt it since the MCDU is primitive.

Discord username (if different from GitHub):
N/A

### PR DESCRIPTION
**Pull request by Watsi01**
Fixes #3013

## Summary of Changes
Removed the check for WP speed constraints on CRZ phase / CRZ speed. Now, on CRZ, any waypoint speed constraint is ignored. CLB and DES speeds will still hold constraints.

## Additional context

Discord username (if different from GitHub): RogePete

## Testing instructions

Test if speed constraints are ignored during cruise.
The below should be fixed now.
```
Example: Mid-flight en route from KORD to KLAS, used following: DCT VORIN DCT IOW DCT CNOTA DCT LNK DCT MCK DCT FQF DCT SINRY DCT CHESZ DCT GGAPP SITEE4, CRZ FL360.

At SINRY (middle of route), MCUD displayed speed constraint of 250K, while managed CRZ was 290. A/C automatically slowed to 250K prior to SINRY and automatically increased back to 290K after passing CHESZ.

Again, at GGAPP where MCDU indicated 210K, A/C dropped to 210K mid cruise once passing. Had to use speed hold mode from thence. Descent wasn't to begin until much later.
```

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

**Review by MisterChocker — APPROVED** [review]
LGTM

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js [hunk 1]
```diff
@@ -1518,7 +1518,7 @@ class FMCMainDisplay extends BaseAirliners {
         if (SimVar.GetSimVarValue("PLANE ALTITUDE", "feet") < 10000) {
             speed = Math.min(speed, 250);
         }
-        return Math.min(speed, this.getSpeedConstraint());
+        return Math.min(speed);
```
Comment (MisterChocker) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js:1521]:
```suggestion
        return speed;
```

**Review by MisterChocker — CHANGES_REQUESTED** [review]

**Comment by Benjozork** [comment]
@MisterChocker do you still have changes requested ?

**Comment by MisterChocker** [comment]
> @MisterChocker do you still have changes requested ?

@Benjozork no, I've committed them myself because I was hoping we don't need any testing for this since the change is pretty straight forward with no possible side effects.

**Review by Benjozork — APPROVED** [review]
LGTM

**Pull request merged**